src/main/java/org/candlepin/subscriptions/tally/facts/FactNormalizer.java

```diff
@@ -329,6 +330,22 @@ private void handleSla(
   private void normalizeQpcFacts(NormalizedFacts normalizedFacts, InventoryHostFacts hostFacts) {
     // Check if this is a RHEL host and set product.
     if (hostFacts.getQpcProducts() != null && hostFacts.getQpcProducts().contains("RHEL")) {
+      if (hostFacts.getSystemProfileArch() != null
+          && CollectionUtils.isEmpty(hostFacts.getSystemProfileProductIds())) {
+        switch (hostFacts.getSystemProfileArch()) {
+          case "x86_64", "i686", "i386":
```

Comment:
One question about these values, is there any documentation or link where to get more information about it? If so, I think it would be really helpful to add a comment here.

Reply:
Hmm I'm not aware of any, @mirekdlugosz do you happen to know of any docs around the system arch we could link to for this?

Reply:
I don't really know, maybe HBI OpenAPI spec file? Or yuptoo.

As I'm on PTO this week, I'm not on position to look for links, sorry. I think it's best to merge as it is and we can add a link in another PR, if we find it.